google/cloud/security/common/data_access/violation_dao.py

```diff
@@ -14,6 +14,7 @@
 
 """Provides the data access object (DAO) for Organizations."""
 
+from collections import namedtuple
```

Comment:
Unrelated to this line, L28 to be exact, can you remove the disables and update accordingly?